api/src/routes/userRouter.js

```diff
@@ -0,0 +1,12 @@
+const Router = require('express');
+
+const passport = require('passport');
+require('../../config/passport')(passport);
+
+const router = new Router();
+const userController = require('../controllers/userController');
```

Comment:
Import {loginUser, Auth } from controller instead of just a controller. And then use it in routes as follows: router.post('login', loginUser)

Reply:
done)